src/RabbitMQMessage.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ
+{
+    public class RabbitMQMessage
+    {
+        public RabbitMQMessage(byte[] body)
```

Comment:
@jeffhollan / @fabiocav - Do you recommend introducing a wrapper type 'RabbitMQMessage'

Reply:
Sorry to chase on this, is it possible to make this adjustment? While I understand this is a API change, I can't imagine many developers could use this library without a meta object, or a way to configure additional details pertaining to and around the message?